app/viewModels/HouseConsignmentViewModel.scala

```diff
@@ -39,6 +39,7 @@ object HouseConsignmentViewModel {
       val helper = new HouseConsignmentAnswersHelper(userAnswers, houseConsignmentIndex)
 
       val sections: Seq[Section] = Seq(
+        helper.headerSection,
```

Comment:
Don't think we need `headerSection`, we can add your `grossMassRow` to the list of rows defined below like `countryOfDestination` and `safetyAndSecurityDetails`.

Might be worth changing this bit of code to make this more apparent:

```
val helper = new HouseConsignmentAnswersHelper(userAnswers, houseConsignmentIndex)

val rows = Seq(
    helper.grossMassRow,
    helper.countryOfDestination,
    helper.safetyAndSecurityDetails
).flatten

val children: Seq[Section] = Seq(
    helper.headerSection,
    helper.departureTransportMeansSection,
    helper.documentSection,
    helper.additionalReferencesSection,
    helper.additionalInformationSection,
    helper.itemSection,
    helper.houseConsignmentConsignorSection,
    helper.houseConsignmentConsigneeSection
)

val houseConsignmentSection: Section = StaticSection(rows = rows, children = children)

HouseConsignmentViewModel(houseConsignmentSection)
```